docker/k8s/README.md

```diff
@@ -23,11 +27,17 @@ in SGX-SIM mode.
     # bootstrap the deployment and service
     kubectl create -f lmdb.yaml
     ```
-4. Start the `EnclaveManager` (to manage SGX enclaves) and `TCS Listener` (to
-   accept and handle requests from clients)  
+4. Start the `EnclaveManager` (to manage SGX enclaves)  
+   For SGX simulator mode, use below command
```

Comment:
s/use below command/use the command below/

Reply:
Done